{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specifications/branches/scenarios.yaml

```diff
@@ -2,4 +2,3 @@ input_draw_count: 10
 random_seed_count: 5
 
 branches:
```

Comment:
Is this intentional? think newlines at the end of files are generally a good idea